api/swagger.yaml

```diff
@@ -1969,11 +1969,37 @@ definitions:
     description: "User modifiable task configuration."
     type: "object"
     properties:
+      PluginSpec:
+        type: "object"
+        properties:
+          Name:
+            description: "The name or 'alias' to use for the plugin. Must not specify with `ContainerSpec`"
+            type: "string"
+          Remote:
+            description: "The plugin image reference to use."
+            type: "string"
+          Disabled:
+            description: "Disable the plugin once scheduled."
+            type: "boolean"
+          PluginPrivilege:
```

Comment:
Yes, we'll then be able to specify all this in entitlements through string arrays and maps.